client/elements/text/sc-segmented-text.html

```diff
@@ -550,22 +579,179 @@ <h3>Could not find the sutta text.</h3>
               return ((!error && loading) || isTextViewHidden);
           }
 
+          _getElememtToHighlight(element) {
+              if (element.id.includes('inline') && !element.id.includes('.')) {
```

Comment:
what does the "id includes dot" check do?

Reply:
if it's segment or paragraph